extension.js

```diff
@@ -1,13 +1,25 @@
-
-const St = imports.gi.St;
 const Main = imports.ui.main;
-const Tweener = imports.ui.tweener;
+const Lang = imports.lang;
+
+let oldSlideIn, hideSliderId;
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).